src/components/Button/Button.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Button.scss';
+
+function Button({ onClick, title }) {
+  return (
+    <button type="button" className={styles.menuButton} onClick={onClick}>
+      {title}
```

Comment:
I'd suggest you to use prop.children approach. That would make your component more flexible. Plus title property usually also needs to be declared separately.